dss/collections/__init__.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+from dss import dynamodb  # type: ignore
+
+
+collection_db_table = f"dss-collections-db-{os.environ['DSS_DEPLOYMENT_STAGE']}"
+
+
+def put_collection(owner: str, uuid: str, permission_level: str = 'owner'):
+    assert permission_level in ('owner', 'read-only')
```

Comment:
`read-only` isn't implemented yet.  It's there for sharing functionality and planning to be compatible since that is coming soon.  I wasn't sure whether to include it, but felt it wouldn't hurt.

Reply:
Also, `read-only` users won't have write functionality obviously, but it's still a put operation into the database with the item containing the user and their permission level for that collection uuid.

Reply:
Removed for now.